integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -114,6 +114,14 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
 
     // 14. finish streaming
     createTable(tableName = "stream_table_finish", streaming = true, withBatchLoad = true)
```

Comment:
please add a test case to *reopen* the stream property after closing stream

Reply:
added